universal-login-commons/src/core/models/wallets/EncryptedWallet.ts

```diff
@@ -1,5 +1,32 @@
 import {asObject, asString, asNumber} from '@restless/sanitizers';
-import {StoredEncryptedWallet, EncryptedWallet, Crypto, CipherParams, KdfParams} from '../../models/wallets/StoredEncryptedWallet';
+
+export interface KdfParams {
```

Comment:
remove export for local interfaces and sanitizers

Reply:
ok